contributing/development-setup.md

```diff
@@ -51,7 +51,20 @@ For more explained information about plugin override check [Development Override
 - Make sure that the PR title follows [*Conventional Commits*](https://www.conventionalcommits.org/).
 - Add comments around your new code that explain what's happening.
 - Commit and push your changes to your branch then submit a pull request against the `master` branch.
-- A repo maintainer will review the your pull request, and may either request additional changes or merge the pull request.
+- A repo maintainer will review the your pull request. 
```

Comment:
A repo maintainer will review your pull request.  (remove "the")

Reply:
done